eahub/profiles/management/commands/sendreactivationemails.py

```diff
@@ -0,0 +1,205 @@
+from email import utils as email_utils
+import sys
+
+from allauth.account import forms
+from allauth.account import utils as account_utils
+from django.contrib.sites import shortcuts
+from django.conf import settings
+from django.core import mail
+from django.core.management import base
+from django.db import models
+from django.template import loader
+
+from ... import models as profiles_models
+
+
+EMEA_COUNTRIES = [
+    "Albania",
+    "Algeria",
+    "Andorra",
+    "Angola",
+    "Armenia",
+    "Austria",
+    "Azerbaijan",
+    "Bahrain",
+    "Belarus",
+    "Belgium",
+    "Benin",
+    "Bosnia and Herzegovina",
+    "Botswana",
+    "Bulgaria",
+    "Burkina Faso",
+    "Burundi",
+    "Cameroon",
+    "Cape Verde",
+    "Central African Republic",
+    "Chad",
+    "Comoros",
+    "Congo - Brazzaville",
+    "Congo - Kinshasa",
+    "Cote d'Ivoire",
+    "Croatia",
+    "Cyprus",
+    "Czech Republic",
+    "Denmark",
+    "Djibouti",
+    "East Germany",
+    "Egypt",
+    "Equatorial Guinea",
+    "Eritrea",
+    "Estonia",
+    "Ethiopia",
+    "Faroe Islands",
+    "Finland",
+    "France",
+    "French Southern Territories",
+    "Gabon",
+    "Gambia",
+    "Georgia",
+    "Germany",
+    "Ghana",
+    "Gibraltar",
+    "Greece",
+    "Greenland",
+    "Guernsey",
+    "Guinea",
+    "Guinea-Bissau",
+    "Hungary",
+    "Iceland",
+    "Iraq",
+    "Ireland",
+    "Isle of Man",
+    "Israel",
+    "Italy",
+    "Jersey",
+    "Jordan",
+    "Kenya",
+    "Kuwait",
+    "Latvia",
+    "Lebanon",
+    "Lesotho",
+    "Liberia",
+    "Libya",
+    "Liechtenstein",
+    "Lithuania",
+    "Luxembourg",
+    "Macedonia",
+    "Madagascar",
+    "Malawi",
+    "Mali",
+    "Malta",
+    "Mauritania",
+    "Mauritius",
+    "Mayotte",
+    "Metropolitan France",
+    "Moldova",
+    "Monaco",
+    "Montenegro",
+    "Morocco",
+    "Mozambique",
+    "Namibia",
+    "Netherlands",
+    "Neutral Zone",
+    "Niger",
+    "Nigeria",
+    "Norway",
+    "Oman",
+    "Palestinian Territories",
+    "People's Democratic Republic of Yemen",
+    "Poland",
+    "Portugal",
+    "Qatar",
+    "Romania",
+    "Russia",
+    "Rwanda",
+    "Réunion",
+    "Saint Helena",
+    "Saint Pierre and Miquelon",
+    "San Marino",
+    "Saudi Arabia",
+    "Senegal",
+    "Serbia",
+    "Serbia and Montenegro",
+    "Seychelles",
+    "Sierra Leone",
+    "Slovakia",
+    "Slovenia",
+    "Somalia",
+    "South Africa",
+    "South Georgia and the South Sandwich Islands",
+    "Spain",
+    "Sudan",
+    "Svalbard and Jan Mayen",
+    "Swaziland",
+    "Sweden",
+    "Switzerland",
+    "Syria",
+    "São Tomé and Príncipe",
+    "Tanzania",
+    "Togo",
+    "Tunisia",
+    "Turkey",
+    "UK",
+    "Uganda",
+    "Ukraine",
+    "Union of Soviet Socialist Republics",
```

Comment:
Doesn't really exist anymore

Reply:
Checked and it doesn't affect us in any way

Reply:
Yes, I was rather amused to see this here, as well as East Germany. This list came from the old Hub.